facades/PC/README.md

```diff
@@ -0,0 +1,99 @@
+                  
+      --[no-]crash-report   Enable crash reporting
+      --create-last-game    Recreates the world of the latest game with a new save file on startup
+      -h, /h, -?, /?, -help, /help, --help
+                            Show help
+      --headless            Start headless (no graphics)
+      --homedir=<homeDir>   Path to home directory
+      --load-last-game      Load the latest game on startup
+      --max-data-size=<size>
+                            Set maximum process data size [Linux only]
+      --oom-score=<score>   Adjust out-of-memory score [Linux only]
+      --override-default-config=<overrideConfigPath>
+                            Override default config
+      --permissive-security
+      --[no-]save-games     Enable new save games
+      --server-port=<serverPort>
+                            Change the server port
+      --[no-]sound          Enable sound
+      --[no-]splash         Enable splash screen
+
+For details, see
+ https://github.com/MovingBlocks/Terasology/wiki/Advanced-Options
+
+Alternatively use our standalone Launcher from
+ https://github.com/MovingBlocks/TerasologyLauncher/releases
+```
+
+### Gradle
+
+The PC Facade provides a couple of Gradle tasks for starting the game on the PC.
+These "Terasology run tasks" offer various default configurations, for instance, to enable debugging or start in _headless mode_.
+
+``` 
+Terasology run tasks
+--------------------
+debug - Run 'Terasology' to play the game as a standard PC application (in debug mode)
+game - Run 'Terasology' to play the game as a standard PC application
+permissiveNatives - Run 'Terasology' with security set to permissive and natives loading a second way (for KComputers)
+profile - Run 'Terasology' to play the game as a standard PC application (with Java FlightRecorder profiling)
+server - Starts a headless multiplayer server with data stored in [project-root]/terasology-server
+```
+
+To pass additional arguments to the PC Facade when running via Gradle, use the `--args` option (see the Gradle [Application Plugin](https://docs.gradle.org/6.9.1/userguide/application_plugin.html) documentation).
+For instance, to print the usage help, run
+
+```sh
+gradlew game --args='--help'
+```
+
+## Architecture
+
+The main entry point for the PC Facade is `Terasology#main`.
+Starting from there, the game engine is initialized and started according to the user inputs.
+
+
+<p align="center">
+<!-- this image is an editable SVG created with Draw.io -->
+<img src="./docs/pc-facade-overview.drawio.svg" alt="Architectural overview of the PC Facade entry point."/>
+</p>
+
+The execution flow is as follows:
+
+1. A `TerasologyEngineBuilder` is instantiated to preparet the game engine.
+
+1. Additional subsystems are collected via the builder. This creates and adds subsystems that are specific for running Terasology on a PC. The facade also chooses respective subsystem implementations depending on whether the game is run in _headless mode_.
```

Comment:
> The facade also chooses respective subsystem implementations depending on whether the game is run in _headless mode_.

It might be worth providing an example of this, for example using `HeadlessGraphics` instead of `LwjglGraphics`.

Reply:
@BenjaminAmos Would a simple
```suggestion
1. Additional subsystems are collected via the builder. This creates and adds subsystems that are specific for running Terasology on a PC. The facade also chooses respective subsystem implementations depending on whether the game is run in _headless mode_. For instance, for a headless server it would choose `HeadlessGraphics` instead of `LwjglGraphics`.
```
suffice or would you like a more detailed example?

Reply:
That level of detail should be enough.